test/TestConfig/makefile

```diff
@@ -100,7 +100,7 @@ endif
 $(info set SRC_ROOT to $(SRC_ROOT))
 
 ifndef BUILD_ROOT
-BUILD_ROOT := $(SRC_ROOT)$(D)..$(D)jvmtest
+BUILD_ROOT := $(SRC_ROOT)$(D)..$(D)test
```

Comment:
This will fail the internal build which uses jvmtest. If you think this BUILD_ROOT is not working, I suppose you can pass a BUILD_ROOT that works for you?

Reply:
Should I instead change the name of the folder in openj9 to jvmtest?

Reply:
Is there a reason that you need to change the BUILD_ROOT? The compiled test material will be placed in the jvmtest folder, and the execution cmd will find them there. All of them should be working by default.

Reply:
The problem is when you clone the repository, compile the jvm, run the test configuration, compile the tests and attempt to run the tests they fail because the name of the folder isnt jvmtest. 

So either the makefile needs to say test, or the folder needs to be called jvmtest or the tests don't run.

Reply:
This is working on our end, please provide the exact command you are using and your output with error? So I can reproduce the issue.

Reply:
After re-cloning and running the full list of instructions from the README.md in the test folder again it seems that the error I thought I was fixing was related to the original problem stated in the issue of not calling `make compile` before calling `make sanity`. 

I'll close this PR and the original issue.

I might create a new issue and PR to make the subsets of `make test` (which does depend on `make compile`) to depend on `make compile` to simplify the test process.

Reply:
I don't think it's necessary to have subsets depend on make compile. All subsets are test targets, in this way we can run ```make compile``` -> ```make sanity``` -> ```make extended``` If you really want to do subtest and compile in one command, you can do ```make compile sanity```